docs/layers.md

```diff
@@ -151,7 +151,7 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8) and Natural E
 
 Polygons from OpenStreetMap representing building footprint, building label_placement points, building_part features, and address points. Starts at zoom 13 by including huge buildings, progressively adding all buildings at zoom 16+. Address points are available at zoom 16+, but marked with `min_zoom: 17` to suggest that they are suitable for display at zoom level 17 and higher.
 
-Individual `building:part` geometries from OSM following the [Simple 3D Buildings](http://wiki.openstreetmap.org/wiki/Simple_3D_Buildings) tags at higher zoom levels are now exported as `building_part` features with specified `kind_detail`.
+Individual `building:part` geometries from OSM following the [Simple 3D Buildings](http://wiki.openstreetmap.org/wiki/Simple_3D_Buildings) tags at higher zoom levels are now exported as `building_part` features with specified `kind_detail`. Building parts will receive a `root_id` that will correspond to the id of the `building` feature that they intersect.
```

Comment:
Pedantic, but: This makes it sound like `building:part`s will _always_ receive a `root_id`, but its possible for `building:part`s to not match any `buildings`. Would it be clearer if it were written as follows?

> Building parts may receive a `root_id` corresponding to the `building` feature, if any, with which they intersect.